pkg/manager/bulkimport/csv.go

```diff
@@ -0,0 +1,37 @@
+package bulkimport
+
+import (
+	"encoding/csv"
+	"io"
+)
+
+type csvParser struct{}
+
+func (csvParser) parse(input io.Reader, fn processFn) error {
+	reader := csv.NewReader(input)
```

Comment:
For importing large csv files setting `ReuseRecord` reduces the allocations a lot. It shouldnt cause any trouble since every row seems to be processed and then discarded anyway.
```golang
	reader := csv.NewReader(input)
  csvreader.ReuseRecord = true
```